pcapps/pyth.cpp

```diff
@@ -801,14 +801,26 @@ int on_upd_test( int argc, char **argv )
 
 int on_upd_price( int argc, char **argv )
 {
-  if ( argc < 2 ) {
+  if ( argc < 5 ) {
```

Comment:
Could this change be made backwards compatible? e.g.

If argc == 2, then `ptr->update()`, else if argc == 5, then `ptr->update(px, ci, st)`, else err

Reply:
Moved to a new subcommand `upd_price_val`